docs/user/getting_started.rst

```diff
@@ -125,6 +125,16 @@ These can be used for debugging and should be stored along with the images to
 allow debugging of kernel problems on devices in the field.
 See :ref:`Debugging <dev-debugging-kernel-oops>` for more information.
 
+.. rubric:: Footnotes
+
+.. [#make_update] ``make update`` only needs to be called again after updating the
+  Gluon repository (using ``git pull`` or similar) or after changing branches,
+  not for each build. Running it more often than necessary is undesirable, as
+  the update will take some time, and may undo manual modifications of the
+  external repositories while developing on Gluon.
```

Comment:
Probably smart to mention that it checks out repositories mentioned in `./modules` and applies patches in `./patches`. Currently this sounds vague, which otoh might have been your intention, since it's part of "Getting Started".

Reply:
I intentionally kept this brief and linked to the more detailed "Working with repositories" section for details instead.